app/services/ServiceUpdateRequest.class.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+const validPaths = {
+  merchantDetails: {
+    name: 'merchant_details/name',
+    addressLine1: 'merchant_details/address_line1',
+    addressLine2: 'merchant_details/address_line2',
+    addressCity: 'merchant_details/address_city',
+    addressPostcode: 'merchant_details/address_postcode',
+    addressCountry: 'merchant_details/address_country',
+    telephoneNumber: 'merchant_details/telephone_number',
+    email: 'merchant_details/email'
+  }
+}
+
+const validOps = {
+  add: 'add',
+  replace: 'replace'
+}
+
+class ServiceUpdateRequestClass {
```

Comment:
If other controllers will be relying on this builder it would be good to have a few unit tests that validate the core functionality.